shoes-swt/spec/shoes/swt/app_spec.rb

```diff
@@ -122,4 +128,20 @@
       it_behaves_like 'reports client area dimensions'
     end
   end
+
+  describe 'App Background color' do
+    it 'should have the given background when specified' do
+      not_resizable = Shoes::Swt::App.new app_unresizable
+      background = not_resizable.shell.background
+      expect(background).to eq swt_salmon
+    end
+
+    # Don't see a way to spec this since it requires a full front end
+    # it 'should have the default system background when unspecified' do
+    #   plain = Shoes::Swt::App.new plain_app
+    #   default_background = ::Swt.display.getSystemColor(::Swt::SWT::COLOR_WIDGET_BACKGROUND)
+    #   background = plain.shell.background
+    #   expect(background).to eq default_background
+    # end
```

Comment:
Why don't you just do a full app? `Shoes::App.new` should be fine, there are also a couple of specs that do that (and one that is called `integration_spec.rb` or soemthing).